src/components/news/NewsCard/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import dayjs from 'dayjs';
+
+import 'src/styles/NewsCard.css';
+
+type tagProps = {
+  id: string;
+  color: 'red' | 'yellow' | 'orange';
+  name: string;
+};
+
+type Props = {
+  date: string;
+  tagList: tagProps[];
+  newsTitle: string;
+  mainText: string;
+};
+
+const NewsCard = ({ date, tagList, newsTitle, mainText }: Props) => (
+  <div className="news-card">
+    <div className="news-card__data-and-tag">
+      <p className="news-card__data-and-tag__date">
+        {dayjs(date.slice(0, 10)).format('YYYY/MM/DD')}
```

Comment:
[imo]
day.jsはかなり強いライブラリなので、sliceしなくても良い感じにしてくれますよ。

```suggestion
       {dayjs(date).format('YYYY/MM/DD')}
```